libqtile/xcbq.py

```diff
@@ -37,6 +37,7 @@
 from __future__ import print_function, division
 
 import six
+import re
```

Comment:
As I said, I'm not an official maintainer, however as you can see the new commit is already visible in this PR, so there's no need to open a new one. tych0 will squash the commits as needed if he thinks it's appropriate.